tests/runtests.py

```diff
@@ -44,31 +53,38 @@ def upath(path):
     'django.contrib.messages',
     'django.contrib.admin',
     'django.contrib.staticfiles',
-    "mezzanine.boot",
-    "mezzanine.conf",
-    "mezzanine.core",
-    "mezzanine.generic",
-    "mezzanine.pages",
-    "mezzanine.forms",
-    'django.contrib.comments',
-    'filebrowser_safe',
-    'grappelli_safe',
+]
+
+if is_mezzanine_installed():
```

Comment:
There's an article somewhere (which I can't find) about `#if ... #else ... #endif` nightmare. Basically what the article is saying `#if ... #else ...` in C exponentially increases the amount of possible code branches which can't be all tested.

[This stackoverflow post summarizes it well](http://stackoverflow.com/a/1851183)